src/Paket.Bootstrapper/Program.cs

```diff
@@ -23,9 +23,13 @@ static void Main(string[] args)
             Console.CancelKeyPress += CancelKeyPressed;
 
             var fileProxy = new FileSystemProxy();
+            var optionsBeforeDependenciesFile = ArgumentParser.ParseArgumentsAndConfigurations(args, ConfigurationManager.AppSettings,
+                Environment.GetEnvironmentVariables(), fileProxy, Enumerable.Empty<string>());
+            ConsoleImpl.Verbosity = optionsBeforeDependenciesFile.Verbosity;
```

Comment:
@vbfox This is what I mean, we call `ParseArgumentsAndConfigurations` again below with an additional argument (which is `Empty` here).

I'm pretty sure we could refactor this somehow, but I couldn't come up with a nice way without changing basically everything.